AstSemantics.md

```diff
@@ -530,3 +530,5 @@ In the MVP, `has_feature` would always return false. As features were added post
 `has_feature` would start returning true. `has_feature` is a pure function, always
 returning the same value for the same string over the lifetime of a single
 instance and other related (as defined by the host environment) instances.
```

Comment:
`has_feature` can't just be a function, if we want to be able to have SIMD types in a function signature. It has to be a structural construct that contains s-expression children.